mne/event.py

```diff
@@ -422,7 +422,7 @@ def find_stim_steps(raw, pad_start=None, pad_stop=None, merge=0,
 
 def _find_events(data, first_samp, verbose=None, output='onset',
                  consecutive='increasing', min_samples=0, mask=None,
-                 uint_cast=False, mask_type='and'):
+                 uint_cast=False, mask_type='and', initial_event=False):
     """Help find events."""
     assert data.shape[0] == 1  # data should be only a row vector
```

Comment:
yes it should be a row vector at this point in the code

Reply:
Ah yes, should have seen this...